### PR TITLE
SCALRCORE-16278 Move go-tfe into go-scalr repo

### PR DESCRIPTION
**Pull request by lbn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,21 +7,21 @@ This is a fork of the Hashicorp Terraform Enterprise Go client.
...
 
 ```
-go get -u github.com/scalr/go-tfe
+go get -u github.com/scalr/go-scalr
 ```
 
 ## Documentation
 
-For complete usage of the API client, see the full [package docs](https://godoc.org/github.com/hashicorp/go-tfe).
+For complete usage of the API client, see the full [package docs](https://godoc.org/github.com/hashicorp/go-scalr).
```
Comment (oliaboo) [inline README.md:15]: hashicorp/go-scalr ? :)

File: workspace_test.go [hunk 2]
```diff
@@ -271,12 +271,12 @@ func TestWorkspacesUpdate(t *testing.T) {
...
-			AutoApply:           Bool(false),
-			Operations:          Bool(false),
-			QueueAllRuns:        Bool(false),
-			TerraformVersion:    String("0.11.1"),
-			WorkingDirectory:    String("baz/"),
+			Name:             String(randomString(t)),
+			AutoApply:        Bool(false),
+			Operations:       Bool(false),
+			QueueAllRuns:     Bool(false),
+			TerraformVersion: String("0.11.1"),
```
Comment (oliaboo) [inline workspace_test.go:278]: is this terraform version ok here?
our minimal version is 0.12.0

File: go.sum [hunk 3]
```diff
@@ -10,8 +10,8 @@ github.com/hashicorp/go-retryablehttp v0.5.2 h1:AoISa4P4IsW0/m4T6St8Yw38gTl5GtBA
 github.com/hashicorp/go-retryablehttp v0.5.2/go.mod h1:9B5zBasrRhHXnJnui7y6sL7es7NDiJgTc6Er0maI1Xs=
 github.com/hashicorp/go-slug v0.4.1 h1:/jAo8dNuLgSImoLXaX7Od7QB4TfYCVPam+OpAt5bZqc=
 github.com/hashicorp/go-slug v0.4.1/go.mod h1:I5tq5Lv0E2xcNXNkmx7BSfzi1PsJ2cNjs3cC3LwyhK8=
-github.com/hashicorp/go-tfe v0.3.31 h1:iVrOwuY2HxEv3IjK3pLpqBEVpfsVkhp2hU9UIFXd1Rs=
-github.com/hashicorp/go-tfe v0.3.31/go.mod h1:DVPSW2ogH+M9W1/i50ASgMht8cHP7NxxK0nrY9aFikQ=
+github.com/hashicorp/go-scalr v0.3.31 h1:iVrOwuY2HxEv3IjK3pLpqBEVpfsVkhp2hU9UIFXd1Rs=
```
Comment (oliaboo) [inline go.sum:13]: go-scalr has no tag v0.3.31

File: README.md (hunk 1 above)
Comment (lbn) [reply README.md]: Done

**Pull request merged**